src/release/main.d

```diff
@@ -569,7 +570,7 @@ ActionList preparePatchRelease ( ref HTTPConnection con, ref Repository repo,
                                         format("Checkout original branch %s",
                                                current_branch.color(mode.bold)));
 
-    scope releaser = new PatchMerger(branches, tags);
+    scope releaser = new PatchMerger(branches, tags, patch_version.metadata);
```

Comment:
`scope`? :suspect:

Reply:
This is not D1 anymore. If you want deterministic destruction, you use structs / `std.typecons.scoped`, otherwise you rely on GC. The fact that in D2 `scope` for classes used to behave the same as in D1 is semi-accidental and is intended to change.